.circleci/config.yml

```diff
@@ -818,7 +818,7 @@ jobs:
     # InitParallel(t, UseExecutor(1))
     # Any tests assigned to an executor greater than the number available automatically use the last executor.
     # Executor indexes start from 0
-    parallelism: 2
+    parallelism: 4
```

Comment:
Unfortunately we have to manually balance tests across the runner - right now we'd still only actually use 2 runners even if 4 are available.

Reply:
Yeah I'm looking into that now - I'll run this one and see what the hot runner is, then rebalance executors across.

Reply:
I think there's limited benefit to more runners because we do have some long running tests which wind up being the long pole.  That may have changed and I think there's at least one test we should reallocate back to runner 0.  I'll put up a PR for that in a minute - it has helped on this PR that Andreas and I are looking at but is mixed in with some other changes that aren't ready to go yet.

Reply:
https://github.com/ethereum-optimism/optimism/pull/9306

Reply:
Ok, sounds good. I'm experimenting to see if having essentially one FP runner per test would help - I'm running that job now. Worst case (e.g. if HTTP takes way too long) we can run a subset of these tests against WS.